x-pack/legacy/plugins/siem/public/components/ml_popover/api.tsx

```diff
@@ -176,63 +219,24 @@ export const stopDatafeeds = async (
 };
 
 /**
- * Fetches Job Details for given jobIds
+ * Fetches a summary of all ML jobs currently installed
  *
- * @param jobIds
- * @param headers
+ * @param signal to cancel request
  */
-export const jobsSummary = async (
-  jobIds: string[],
-  headers: Record<string, string | undefined>,
-  signal: AbortSignal
-): Promise<Job[]> => {
+export const getJobsSummary = async (signal: AbortSignal): Promise<JobSummary[]> => {
   const [kbnVersion] = useKibanaUiSetting(DEFAULT_KBN_VERSION);
   const response = await fetch(`${chrome.getBasePath()}/api/ml/jobs/jobs_summary`, {
     method: 'POST',
     credentials: 'same-origin',
-    body: JSON.stringify({ jobIds }),
+    body: JSON.stringify({}),
     headers: {
       'content-type': 'application/json',
-      'kbn-xsrf': kbnVersion,
       'kbn-system-api': 'true',
-      ...headers,
+      'kbn-version': kbnVersion,
+      'kbn-xsrf': kbnVersion,
```

Comment:
Uh oh....This is not good.

It should be this right?
```
'kbn-xsrf': chrome.getXsrfToken()
```

Looks like at some point we started using `kbnVersion` for some reason and then it just kind of spread quickly out of hand.

Reply:
This is okay actually, and was a result of re-bootstrapping the app in preparation for migrating to the new platform (and supporting embeddables :).

The actual implementation of `chrome.getXsrfToken()` will return the kbnVersion just as this does, and while I agree we shouldn't stray from using the abstraction, this instance is approved while we migrate to the new Kibana platform.

https://github.com/elastic/kibana/blob/34cd8a3c6738fdd3b1432e5c324bbe31c433299b/src/legacy/ui/public/chrome/api/xsrf.js#L20-L24